languagetool-language-modules/eo/src/main/resources/org/languagetool/rules/eo/grammar.xml

```diff
@@ -4047,6 +4080,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         <example correction="simplan|facilan">Li donis <marker>malkomplikan</marker> respondon.</example>
       </rule>
     </rulegroup>
+    <rulegroup id="HUMIDA" name="humida → malseka">
+      <rule>
+        <pattern>
+          <marker><token inflected="yes" regexp="yes">(ne|mal)?humid((umi|igi|iĝi)|(a|o|eco|ega|eta)j?n?)</token></marker>
+        </pattern>
+        <message>La vorto ‘humida’ estas ne estas oficiala esperanta vorto, kaj la vorto ‘malseka’ normale signifas ekzakte la saman. Ĉu vi intencis ‘<suggestion><match no="1" regexp_match="humid" regexp_replace="malsek"/></suggestion>’?</message>
```

Comment:
Good point for the quotation mark. I will change this.

We can remove this rule. It was just a general thought that I had that I was starting to hear “humida” at places were “malseka” should generally be used. It is probably out of the scope of the tool to know whether we are in a scientific context or not, so removing this rule seems acceptable.